src/main/java/uk/ac/sanger/sccp/stan/service/DnapStudyService.java

```diff
@@ -0,0 +1,32 @@
+package uk.ac.sanger.sccp.stan.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+import uk.ac.sanger.sccp.stan.model.DnapStudy;
+import uk.ac.sanger.sccp.stan.repo.DnapStudyRepo;
+
+import java.util.Optional;
+
+/**
+ * Service dealing with {@link DnapStudy studies}
+ * @author dr6
+ */
+@Service
+public class DnapStudyService extends BaseAdminService<DnapStudy, DnapStudyRepo> {
+    @Autowired
+    public DnapStudyService(DnapStudyRepo dnapStudyRepo,
+                            @Qualifier("dnapStudyNameValidator") Validator<String> nameValidator) {
+        super(dnapStudyRepo, "DNAP study", "Name", nameValidator);
+    }
+
+    @Override
+    protected DnapStudy newEntity(String name) {
```

Comment:
Do we need a validation for dnap study name ( as Katy mentioned it will be a 6-7 digit code followed by "-" then a project description.) somewhere?

Reply:
I don't think so. We do not rely on it being any particular format.

Reply:
Ok